Classes/Aspect/CommandControllerAspect.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Netlogix\Cli\IncreaseDatabaseTimeout\Aspect;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\ORM\EntityManagerInterface;
+use Exception;
+use Neos\Flow\Annotations as Flow;
+use Neos\Flow\Aop\JoinPointInterface;
+
+/**
+ * @Flow\Aspect
+ */
+class CommandControllerAspect
+{
+	/**
+	 * @Flow\InjectConfiguration(path="timeouts")
+	 * @var array
+	 */
+	protected $timeouts = [];
+
+	/**
+	 * @Flow\Inject
+	 * @var EntityManagerInterface
+	 */
+	protected $entityManager;
+
+	/**
+	 * @Flow\Before("method(Neos\Flow\Cli\CommandController->processRequest())")
+	 * @param JoinPointInterface $joinPoint
+	 * @return void
+	 * @throws \Doctrine\DBAL\Exception
+	 */
+	public function increaseDatabaseTimeout(JoinPointInterface $joinPoint)
+	{
+		$commandIdentifier = $joinPoint->getMethodArgument('request')
+			->getCommand()
+			->getCommandIdentifier();
+
+		$matchCount = preg_match('/[^:]*:(?<commandName>.*)/', $commandIdentifier, $matches);
+
+		if ($matchCount === 0 || !array_key_exists($matches['commandName'], $this->timeouts)) {
```

Comment:
$matchCount could be false here

Reply:
Glad you noticed, I've posted a bug fix to make sure this is handled correctly.